### PR TITLE
[0.10] - Restore check-invalid-reads

### PR DESCRIPTION
**Pull request by niden**
Added verbiage for `check-invalid-reads`

### CODE REVIEW

**Comment by niden** [comment]
cc @sergeyklay  @danhunsaker Input please

File: en/optimizations.md [hunk 1]
```diff
@@ -29,6 +29,67 @@ This pass counts how many times a function or method is called within the same m
...
+
+
+```zep
+namespace Acme;
+
+class ForInRange
+{
+    public static function forEmpty(var n)
+    {
+        var i = NULL;
```
Comment (sergeyklay) [inline en/optimizations.md]:
```suggestion
        var i = null;
```

**Review by sergeyklay — CHANGES_REQUESTED** [review]

**Review by danhunsaker — COMMENTED** [review]
Nothing to add, other than to ask whether we just removed this one, recently, and if so, whether the old verbiage was used as a starting point, here?

**Comment by niden** [comment]
@danhunsaker We removed it by mistake (my bad). The old verbiage was really the first sentence. I added a bit more to explain what this option does.

**Pull request merged**